src/PrestaShopBundle/Resources/config/routing/admin/sell/customer_service/merchandise_return.yml

```diff
@@ -23,3 +23,27 @@ admin_merchandise_returns_filter:
     redirectRoute: admin_merchandise_returns_index
     _legacy_controller: AdminReturn
 #    _legacy_link: AdminReturn:submitFilterorder_return
+
+admin_order_returns_edit:
+    path: /{orderReturnId}/edit
+    methods: [GET]
+    defaults:
+        _controller: PrestaShopBundle:Admin/Sell/CustomerService/MerchandiseReturn:edit
+        _legacy_controller: AdminReturn
+    #    _legacy_link: AdminReturn:updateorder_return
+    #    _legacy_parameters:
+    #      id_order_return: orderReturnId
+    requirements:
+        orderReturnId: \d+
+
+admin_order_returns_update:
```

Comment:
This route is a duplicate of `admin_order_returns_edit` I don't understand its purpose

Reply:
Oh I get it, it's because one is GET and the other is POST, if you had two distinct actions in your controller why not But I think you can fuse them and allow `[GET, POST]` as methods